lib/matplotlib/cbook.py

```diff
@@ -708,6 +708,9 @@ def is_string_like(obj):
         return False
     return True
 
+def is_list_like(obj):
```

Comment:
I will move this into `testing/__init__.py`  it is the only place that this is currently used and we can pull it up to cbook later if needed.